Scripts/Bootstrap.ps1

```diff
@@ -0,0 +1,81 @@
+﻿param (
+    [Parameter(Mandatory=$true)]
+    [string]$NuGetDropPath,
+    [Parameter(Mandatory=$true)]
+    [string]$FuncTestRoot)
```

Comment:
Actually, I prefer that we do not have default values, since we have accurate control over the scripts especially on the CI.